submissions/loisAdegbohungbe/HTML/product.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Maxi 4-Burner Gas Cooker | Louissa Store</title>
+</head>
+<body>
+    <h2>Maxi 4-Burner Gas Cooker</h2>
+
+    <!--This section is for product images and price-->
+    <section>
+        <div>
+            <img src="images/image1.jpg" width="300" alt="image1"/>
+            <img src="images/image2.jpg" width="300" alt="image2"/>
+        </div>
+        <h1>Price: &#8358;50,400</h1>
+    </section>
+
+    <!--This section is for product description-->
+    <section>
+        <h3>Product Description</h3>
+        <p>Maxi 4-Burner is a omfortable and easy to use gas cooker. 
+            Cook your meals faster with this appliance and enjoy neatness and ease.
+            Its durability is sure and getting this, you have made a rich step to cooking your delicious meals.
+        </p>
+    </section>
+
+    <!--This navigation section displays other related products-->
+    <nav>
```

Comment:
Why are you using a nav tag here? Nav tags are used for major blocks of navigation links. You can use a section tag instead